cdc/owner.go

```diff
@@ -432,16 +432,29 @@ func (o *ownerImpl) newChangeFeed(id model.ChangeFeedID, processorsInfos model.P
 
 	ddlHandler := newDDLHandler(o.pdClient, checkpointTs)
 
+	existingTables := make(map[uint64]struct{})
+	for _, subCfInfo := range processorsInfos {
+		for _, tbl := range subCfInfo.TableInfos {
+			existingTables[tbl.ID] = struct{}{}
+		}
+	}
+
 	tables := make(map[uint64]schema.TableName)
 	orphanTables := make(map[uint64]model.ProcessTableInfo)
-	for id, table := range schemaStorage.CloneTables() {
+	for tid, table := range schemaStorage.CloneTables() {
 		if detail.ShouldIgnoreTable(table.Schema, table.Table) {
 			continue
 		}
 
-		tables[id] = table
-		orphanTables[id] = model.ProcessTableInfo{
-			ID:      id,
+		tables[tid] = table
+		if _, ok := existingTables[tid]; ok {
+			log.Info("ignore existing replication table",
```

Comment:
No need to log this? Or maybe use the `DEBUG` level.

Reply:
Ignore a table may be a sensitive operation, if we skip a table by mistake, data loss happens. Still keep the `INFO` level, but add a `checkpointTs` field in log

Reply:
In the case this PR aimed to fix, it seems this would output one line for each table involved. If we are to keep it, we should strive to make it as simple as possible.

For example:

1. Use shorter message like "Ignore known table"
1. Remove fields that don't change during the iteration, here we can log `changefeedID` and `checkpoint` outside the loop
1. Maybe shorter field names like "tid" and "tbl"

Reply:
change to `DEBUG` level. `checkpoint` is not always the same. `changefeedID` well is logged outside already.